libvips/foreign/dzsave.c

```diff
@@ -303,7 +303,7 @@ struct _VipsForeignSaveDz {
 
 	/* Count zoomify tiles we write.
 	 */
-	int tile_count;
+	int tile_count; // (atomic)
```

Comment:
Does this need to be marked as `volatile`? The advice at https://docs.gtk.org/glib/func.atomic_int_get.html is a little ambiguous.

Reply:
Ha ha, thank you, the history of `volatile` in C has been, er, volatile.